charts/civil-ccd/values.preview.template.yaml

```diff
@@ -1,6 +1,7 @@
 civil-service:
   java:
     ingressHost: ${SERVICE_NAME}.preview.platform.hmcts.net
+    image: 'hmctspublic.azurecr.io/civil/service:pr-1843'
```

Comment:
@filippo-possenti-version1 This will need to be reverted before merging as well.